RestSharp/Extensions/StringExtensions.cs

```diff
@@ -281,7 +281,7 @@ public static string ToPascalCase(this string text, bool removeUnderscores, Cult
                             restOfWord = restOfWord.ToLower(culture);
                         }
 
-                        char firstChar = char.ToUpper(word[0], culture);
+                        char firstChar = culture.TextInfo.ToUpper(word[0]);
```

Comment:
`ToUpper` and `ToLower` methods taking `CultureInfo` as an argument are not available in UWP. I've just changed this call to the internal implementation of `ToUpper`.